packs.json

```diff
@@ -386,6 +386,15 @@
     "position": 1,
     "size": 20
   },
+  {
+    "code": "ph",
+    "cycle_code": "borealis",
+    "date_release": "2022-12-12",
```

Comment:
Should this be the 12th or the 9th?

Reply:
confirmed we are going with the digital release date of the 9th. thanks for flagging!